superset-frontend/src/common/components/index.tsx

```diff
@@ -28,6 +28,7 @@ import { DropDownProps } from 'antd/lib/dropdown';
  */
 // eslint-disable-next-line no-restricted-imports
 export {
+  Alert,
```

Comment:
Wanted to use Antd Alert, but it looks inconsistent with the error box in chart container.